sdcm/nemesis.py

```diff
@@ -393,15 +393,16 @@ def get_list_of_methods_by_flags(
         return disrupt_methods_list
 
     def get_list_of_subclasses_by_property_name(self, list_of_properties_to_include):
-        flags = {flag_name: True for flag_name in list_of_properties_to_include}
+        flags = {flag_name.strip('!'): not flag_name.startswith(
```

Comment:
it's not clear why this `strip(!)` is needed.

Reply:
since the key of this dict, is the name of the flag (hence we don't want the ! to be part of it)
i.e. :

```yaml
nemesis_selector: !disabled,topology_changes
``` 
means that: 

```python
list_of_properties_to_include=["!disabled", "topology_changes"]
```